framework/src/main/java/framework/db/Configuration.java

```diff
@@ -0,0 +1,60 @@
+package main.java.framework.db;
+
+/**
+ * Database configuration
+ * ToDo: parse Sonarqube confirutaion file
+ * ToDo: test with other DBMS
+ *
+ * @author Klara Erlebachova
+ */
+public class Configuration {
+
+    public static final Configuration INSTANCE = new Configuration();
+
+
+    /** URL of SonarQube DB */
+    private String connectionString = "jdbc:h2:tcp://localhost:9092/sonar";
+    /** JDBC driver used */
+    private String jdbcDriver = "org.h2.Driver";
+    /** User name used for connection */
+    private String dbUser = "";
+    /** Password for DB_USER */
+    private String dbPassword = "";
+
+    public Configuration() {
```

Comment:
Nechala jsem to tam pro co nejsnazší testování, počítám s tím, že až budeš dělat čtení configu, tak se tohle změní